src/main/java/net/atos/zac/configuratie/ConfiguratieService.java

```diff
@@ -61,15 +61,14 @@ public class ConfiguratieService {
 
     public static final String ENV_VAR_ZGW_API_CLIENT_MP_REST_URL = "ZGW_API_CLIENT_MP_REST_URL";
 
+    public static final Integer MAX_FILE_SIZE_MB = 80;
```

Comment:
Nice. Ik zou hier nog wel een Javadoc commentje bij zetten en verwijzen naar die andere property in WildFly. Voor de toekomst wel fijn.